examples/xgboost/job_config_gen.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# change to "gpu_hist" for gpu training
+TREE_METHOD="hist"
+
+prepare_job_config() {
+    python3 utils/prepare_job_config.py --site_num "$1" --training_mode "$2" --split_method "$3" \
+    --lr_mode "$4" --nthread 16 --tree_method "$5"
+}
+
+prepare_job_config 5 bagging exponential scaled $TREE_METHOD
```

Comment:
Some of these scripts and files (like this one) apply only to tree based approach so should they be at top level vs in the tree-based folder?

Reply:
good question, I've made the prepare_job_config generalize for both approach.

So this one should be at top level